content/en/tests/setup/javascript.md

```diff
@@ -435,13 +473,70 @@ If the browser application being tested is instrumented using [Browser Monitorin
 [8]: /continuous_integration/guides/rum_integration/
 [9]: https://docs.cypress.io/guides/references/migration-guide#Migrating-to-Cypress-100
 [10]: https://docs.cypress.io/api/plugins/after-run-api
+[11]: https://docs.cypress.io/api/plugins/after-spec-api
 {{% /tab %}}
 
 {{< /tabs >}}
 
-### Using Yarn 2 or later
+### How to fix "Cannot find module 'dd-trace/ci/init'" errors
+
+When using `dd-trace` you might encounter this error message:
+
+```text
+ Error: Cannot find module 'dd-trace/ci/init'
+```
+
+This might be because of an incorrect usage of `NODE_OPTIONS`.
+
+For example, if your GitHub Action looks like this:
+```yml
+jobs:
+  my-job:
+    name: Run tests
+    runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: -r dd-trace/ci/init
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install node
+        uses: actions/setup-node@v3
+      - name: Install dependencies
+        run: npm install
+      - name: Run tests
+        run: npm test
+```
+
+This will **not** work because `NODE_OPTIONS` will be interpreted by every node process, including `npm install`. If you try to import `dd-trace/ci/init` before it's intalled, this step will fail.
```

Comment:
```suggestion
**Note:** This does not work because `NODE_OPTIONS` are interpreted by every node process, including `npm install`. If you try to import `dd-trace/ci/init` before it's installed, this step fails.
```